en.txt

```diff
@@ -0,0 +1,67 @@
+About me:
+	I mostly have experience with Linux servers, Unix, c programming language
+	I like Implementing interesting ideas.
+	I have worked briefly in 2 startups, here is what I did:
+	Start-up 1 - inn0web, website: https://inn0web.com, (time period 2022.08 - 2022.10):
+		- I wrote a bot for finding bicycles of a specific price, in particular locations.
+		When found, send information to Telegram.
+		- The bot is designed to find the most popular Reddit posts, and according to the topic, make 10 min
+		video montage to upload them to different YouTube channels
+	Start-up 2 - Monopulse, website: https://www.monopulse.eu, (time period ~2023.02): 
+		- I was assigned a task to work with embedded Linux in the yacto project.	
+	~2023.09 I was selling WordPress installation services. 
+	Service to transfer all Etsy products to WordPress using Etsy & Woocommerce API
+
+	NOTE: The source code of these projects is not public. but can be requested via adomasda@gmail.com
+
+Java:
+	I worked with a professional Java mentor who assigned me Java tasks, 
+	which were reviewed, corrected, and completed through a process involving 
+	Jira for task management, Git for version control, and code reviews
+	conducted via Pull Requests. 
+	You can see the entire tasking process in the GitHub links below.
+		https://github.com/Adomo-Software/java-fix-me
+			Tasks: JFM-1, JFM-2, JFM-4
+		https://github.com/Adomo-Software/rating-system
+			Tasks: JC-2, JC-4
+		https://github.com/Adomo-Software/caesars-palace-casino
+			Tasks: JBH-17
+		https://github.com/Adomo-Software/cherry-tree-hotel
+			Tasks: JBH-13
+		https://github.com/Adomo-Software/continental-hotel
+			Tasks: JBH-5
+		https://github.com/Adomo-Software/hotel
+			Tasks: JBH-1, JBH-4
```

Comment:
Under every or in general you nmust mention what you have learned, and what themes you went thru in Java;